src/Command/App/NewCommand.php

```diff
@@ -36,13 +34,14 @@ protected function configure(): void {
    * @throws \Exception
    */
   protected function execute(InputInterface $input, OutputInterface $output): int {
-    $this->output->writeln('Acquia recommends most customers use <options=bold>acquia/drupal-recommended-project</>, which includes useful utilities such as Acquia Connector.');
-    $this->output->writeln('<options=bold>acquia/drupal-minimal-project</> is the most minimal application that will run on the Cloud Platform.');
+    $this->output->writeln('Acquia recommends most customers use <options=bold>acquia/drupal-recommended-project</> to setup a Drupal project, which includes useful utilities such as Acquia Connector.');
```

Comment:
Looks like we don't assert that this help text gets printed, can you add an assertion for that to the NewCommand test case?

You don't need to copy the whole string, just the first few words if you want.

```
    $this->assertStringContainsString('Acquia recommends most customers', $output);
    $this->assertStringContainsString('<options=bold>acquia/next-acms</> is a starter', $output);
```

Reply:
Added both (one in drupal test and one in nextjs test)